src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
Don’t use inheritance here. if you want to use some method of UserService you should create an instance 
`private final UserService service = new UserService();`